scripts/reset_iks_api_key.sh

```diff
@@ -61,16 +64,16 @@ fetch_data
 if [ "${reset}" == true ]; then
     if [ "$PRIVATE_ENV" = true ]; then
         if [ "$CLUSTER_ENDPOINT" == "private" ] || [ "$CLUSTER_ENDPOINT" == "default" ]; then
-            RESET_URL="https://private.$REGION.containers.$CLOUD_ENDPOINT/v1/keys"
+            RESET_URL="https://private.$REGION.$IBMCLOUD_CS_API_ENDPOINT/v1/keys"
             result=$(curl -i -H "accept: application/json" -H "Authorization: $IAM_TOKEN" -H "X-Auth-Resource-Group: $RESOURCE_GROUP_ID" -X POST "$RESET_URL" 2>/dev/null)
             status_code=$(echo "$result" | head -n 1 | cut -d$' ' -f2)
         elif [ "$CLUSTER_ENDPOINT" == "vpe" ]; then
-            RESET_URL="https://api.$REGION.containers.$CLOUD_ENDPOINT/v1/keys"
+            RESET_URL="https://api.$REGION.$IBMCLOUD_CS_API_ENDPOINT/v1/keys"
```

Comment:
I don't know if all override API endpoints support region, will need to test this.